app/assets/stylesheets/task.scss

```diff
@@ -21,3 +30,17 @@
 
   i { line-height: 1rem; }
 }
+
+.tag {
+  @extend .chip;
+  background-color: #E6E18F;
```

Comment:
Color literals like `#E6E18F` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6E18F` should be written as `#e6e18f`